bibliopixel/drivers/driver_base.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, num=0, width=0, height=0, c_order=ChannelOrder.RGB,
         self.lastUpdate = 0
         self.brightness_lock = threading.Lock()
         self._brightness = 255
+        self._waiting_brightness = None
```

Comment:
Yeah... I really don't know why it wasn't. It would throw an exception if you didn't set the brightness before running animations on some drivers. It's the one thing I hate about Python that you can use vars that were never initialized previously.